cloud-common/src/main/java/com/sequenceiq/cloudbreak/domain/SecretProxy.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.cloudbreak.domain;
+
+import com.sequenceiq.cloudbreak.service.secret.SecretService;
+
+public class SecretProxy extends Secret {
+
+    private static final long serialVersionUID = 1L;
+
+    private final transient SecretService securityService;
```

Comment:
Should be called secretservice

Reply:
Not suggest, because this is a proxy which is replaces the original secret in runtime and changes it's behavior like Hibernate proxyes.